pkg/adapter/github/adapter.go

```diff
@@ -21,10 +21,11 @@ import (
 	"fmt"
 	"log"
 	"net/http"
+	"strconv"
+	"strings"
 
-	"github.com/cloudevents/sdk-go/pkg/cloudevents"
+	"github.com/cloudevents/sdk-go"
 	"github.com/cloudevents/sdk-go/pkg/cloudevents/client"
```

Comment:
you just need `cloudevents.Client` from the root. you can delete this line.

Reply:
Done, thanks!